app/_hub/kong-inc/request-termination/_index.md

```diff
@@ -55,7 +55,7 @@ params:
       minimum_version: "2.6.x"
       required: false
       default: false
-      description: 'When set, the plugin will echo a copy of the request back to the client. The main usecase for this is debugging. It can be combined with `trigger` in order to debug requests on live systems without disturbing real traffic.'
+      description: 'When set, the plugin will echo a copy of the request, the router kong and the kong instance information back to the client. The main usecase for this is debugging. It can be combined with `trigger` in order to debug requests on live systems without disturbing real traffic.'
```

Comment:
```suggestion
      description: 'When set, the plugin will echo a copy of the request, the router information, and the kong instance information back to the client. The main usecase for this is debugging. It can be combined with `trigger` in order to debug requests on live systems without disturbing real traffic.'
```